hw_diag/utilities/hardware.py

```diff
@@ -250,12 +250,19 @@ def lora_module_test():
 
 
 def get_public_keys_and_ignore_errors():
-    public_keys = get_public_keys_rust()
-    if not public_keys:
-        error_msg = "ECC failure"
+    try:
+        public_keys = get_public_keys_rust()
+        if not public_keys:
+            error_msg = "ECC failure"
```

Comment:
nit: might as well define this once at beginning of method so you can reuse it in the `except` clause too

Reply:
Done! 😊

Reply:
@marvinmarnold Just going to test this to make sure the page looks good when ECC failure occurs